src/components/learner-credit-management/data/hooks/useSuccessfulReminderToastContextValue.js

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useMemo, useState } from 'react';
+
+const generateSuccessRemindMessage = (assignmentUuidCount) => {
+  if (assignmentUuidCount > 1) {
+    return `Reminders sent (${assignmentUuidCount})`;
+  }
+  return 'Reminder sent';
+};
+
+const useSuccessfulReminderToastContextValue = () => {
+  const [isToastOpen, setIsToastOpen] = useState(false);
+  const [assignmentUuidCount, setAssignmentUuidCount] = useState();
+
+  const handleDisplayToast = useCallback((assignmentUuids) => {
+    setIsToastOpen(true);
+    setAssignmentUuidCount(assignmentUuids);
+  }, []);
+
+  const handleCloseToast = useCallback(() => {
+    setIsToastOpen(false);
```

Comment:
[suggestion]: I think we should a unit for this. There's an example in `BudgetDetailPageWrapper.tests.jsx` that tests  something similar for the cancel toast.